en/reference/assertions.md

```diff
@@ -29,58 +32,612 @@ sign-key-sha3-384: <key id> # Encoded key id of signing key
+aPvkRw3XCVeYq0K6w9GRsk1YFErD4XmXXZjDYY650MX9v42Sz5MmphHV8jdIY5ssbadwFSe2rCQI
+6UX08zy7RsIb19hTndE6ncvSNDChUR9eEnCm73eYaWTWTnq1cxdVP/s52r8uss++OYOkPWqh5nOu
+haRn7INjH/yZX4qXjNXlTjo0PnHH0q08vNKDwLhxS+D9du+70FeacXFyLIbcWllSbJ7DmbumGpFo
+yYbtj3FDDPzachFQdIG3lSt+cSUGeyfSs6wVtc3cIPka/2Urx7RprfmoWSI6+a5NcLdj0u2z8O96
+HxeIgxDpg/3gT8ZIuFKePMcLDM19Fh/p0ysCsX+84B9chNWtsMSmIaE57V+959MVtsLu7SLb9gi7
+skrju0pQCwsu2wHMLTNd1f3PTHmrr49hxetTus07HSQUApMtAGKzQilF5zqFjbyaTd4xgQbd+PKW
+CjFyzQTDOcUhXpuUGt/IzlqiFfsCsmbj2K4KdSNYMlqIgZ3Azu8KvZLIhsyN7v5vNIZSPfEbjdeu
+ClU9r0VRiJmtYBUjcSghD9LWn+yRLwOxhfQVjm0cBwIt5R/yPF/qC76yIVuWUtM5Y2/zJR1J8OFq
+qWchvlImHtvDzS9FQeLyzJAOjvZ2CnWp2gILgUz0WQdOk1Dq8ax7KS9BQ42zxw9EZAEPw3PEFqRy
+IQsRTONp+iVS8YxSmoYZjDlCgRMWUmawez/Fv5b9Fb/XkO5Eq4e+KfrpUujXItaipb+tV8h5v3tr
+oG3Ie3WOHrVjCLXIdYslpL1O4nadqR6Xv58pHj6k
+```
 
 ## model
 
 The model assertion is a statement by a brand about the properties of a device model.
 
+The format is as follows:
+
+```text
+type:              model
+authority-id:      <authority account id>
+revision:          <int>
+series             <string>
+brand-id           <account id>
+model              <model id>
+classic            <true|false>
+store              <string>
+display-name       <descriptive string>
+architecture       <debian architecture name>
+gadget             <gadget snap name>
+kernel             <kernel snap name>
+required-snaps     <string>
+timestamp          <UTC datetime>
+sign-key-sha3-384: <key id> # Encoded key id of signing key
+
+<signature>                 # Encoded signature
+```
+
+The index for this assertion is the tuple <`series`, `brand-id`, `model`>. `series`
+allows the brand to define which release of the platform the device uses.
+“rolling” is the name of the development series that bridges stable series,
+which have names like “16” or “18”. `brand-id` is the account id of the brand, and `model`
+is a string that identifies a set of devices as desired by the brand.
+
+The (optional) `classic` flag tells us if this is an all-snap system (false) or not (true).
+If not set, `architecture`, `gadget`, and `kernel` are mandatory. If set, `kernel`
+is forbidden and `architecture` and `gadget` are optional. If not present, an
+all-snap system is assumed.
+
+The optional `store` header specifies a particular branded virtual store to be
+used for this model.
+The store in question has custom content specific to that model, curated by that brand,
+and managed by the store owner or the relevant brand. Each model knows which
+store it needs to speak to in order to get the appropriate content. If left out
+the device defaults to use the main Ubuntu store.
+
+`gadget` and `kernel` specify respectively the gadget and kernel snaps used to build
+the device images. `required-snaps` (optional) is a list of snaps that must be installed at
+all times, and cannot be removed from the device.
+
+For instance, this is kvm's Ubuntu Core model assertion:
```

Comment:
KVM? This is the std. model assertion for any amd64 based device, nothing specific for KVM.